ArduCopter/switches.cpp

```diff
@@ -4,13 +4,13 @@
 
 //Documentation of Aux Switch Flags:
 struct {
-    uint8_t CH6_flag    : 2; // 0, 1    // ch6 aux switch : 0 is low or false, 1 is center or true, 2 is high
```

Comment:
This change looks unrelated?

Reply:
@peterbarker, I had to do that actually or the read_3pos_switch wouldn't accept the .CH6_flag as an argument.

Txs for reviewing!